R/data-specs.R

```diff
@@ -180,3 +190,7 @@
 #' }
 #' @source Generated from `reportingResults_country.csv` dataset in the `gsm` package.
 "reportingResults_country"
+
+df_dim_desc <- function(df){
```

Comment:
Felt like a good idea to have the dimensions programmatically render the dimensions of the dataframe statement